mava/utils/make_env.py

```diff
@@ -68,19 +68,18 @@ def make_jumanji_env(
         A tuple of the environments.
     """
     # Config generator and select the wrapper.
-    task_config = jumanji_registry.get_task_config(
-        config["env"]["env_name"], config["env"]["scenario"]
-    )
-    config["env"]["scenario"] = {config["env"]["scenario"]: task_config}
+    task_config = jumanji_registry.get_task_config(config.env.env_name, config.env.scenario)
+    config.env.scenario = {config.env.scenario: task_config}
 
-    generator = _jumanji_env_class[env_name]["generator"]
+    generator = _jumanji_wrappers_and_args[env_name]["generator"]
     generator = generator(**task_config)
-    wrapper = _jumanji_env_class[env_name]["wrapper"]
+    wrapper = _jumanji_wrappers_and_args[env_name]["wrapper"]
 
     # Create envs.
-    env = jumanji.make(env_name, generator=generator)
+    time_limit = config.env.time_limit
+    env = jumanji.make(env_name, generator=generator, time_limit=time_limit)
     env = wrapper(env)
-    eval_env = jumanji.make(env_name, generator=generator)
+    eval_env = jumanji.make(env_name, generator=generator, time_limit=time_limit)
```

Comment:
Hmmmm I worry that not all jumanji envs will have this argument, can you double check that both rware and lbf have and then I'm happy to leave it for now until we find an env that doesn't have it

Reply:
I checked a few Jumanjii envs that we're aiming to support in the future and found that only the CVRP among all of them doesn't provide this attribute but all is good 🥳

Reply:
This issue will be resolved as soon as @WiemKhlifi 
 incorporates a time limit into the kwargs in the yaml file, followed by utilizing `jumanji.make(env_name, generator=generator, **kwargs)`

Reply:
@WiemKhlifi I reopened the  conversation in case you will make the changes I suggested :sweat_smile: